File: sololearn/Roadrunner/Roadrunner.py
#!/usr/bin/python

dist = int(input())
v_r = int(input())
v_c = int(input())

time_r = dist / v_r
time_c = 50 / v_c

if time_c > time_r:
    print("Meep Meep")
else:
    print("Yum")

File: sololearn/Roadrunner/Roadrunner.cpp
/*
Author: chankruze (chankruze@geekofia.in)
Created: Wed Aug 26 2020 04:15:24 GMT+0530 (India Standard Time)

Copyright (c) Geekofia 2020 and beyond
*/

#include <iostream>

using namespace std;

int main(int argc, char const *argv[]) {
    int dist, v_r, v_c;
    cin >> dist;
    cin >> v_r;
    cin >> v_c;

    if ((double)(50.00 / v_c) > (double)(dist / v_r)) {
        cout << "Meep Meep" << endl;
    } else {
        cout << "Yum" << endl;
    }
}
